### DISCUSSION TITLE
Kindle lookups - does it work for Kindle Scribe?

### DISCUSSION THREAD

**Discussion by contributor** [FreeLanguageTools/vocabsieve#155] [General]
In "Select your Kindle root (top-level) directory" Kindle Scribe is not listed.

It does listed in Windows Explorer (Windows 11) under This PC:

![image](https://github.com/FreeLanguageTools/vocabsieve/assets/598873/1de2c8f4-caa7-4a73-a8f4-f1bac1c8b9e6)

But in Vocabcieve, kindle item is not present

**Comment by 1over137** [comment]
no idea, maybe it has a different folder structure

**Reply by contributor** [reply]
I think the problem is that "Select directory" dialog (tried on Windows 11 and Windows 10) is not able to select Kindle at all. For example, if I try to open a directory in vscode or notepad++ - there is no kindle item there. However, if I try to open a file in those programs - I can see kindle content, and I can select any file to open.

This leads me to believe that windows doesn't consider Kindle to be a directory/folder.

**Reply by 1over137** [reply]
try copying everything from the device to a folder and select that to see if it works

**Reply by contributor** [reply]
thank you. Yes, when copied to a folder - it works.
